src/ducks/registry/index.js

```diff
@@ -25,4 +27,24 @@ export const getRegistryKonnector = (state, slug) => state.registry.konnectors.f
 
 // action creators sync
 
-export const initializeRegistry = konnectors => ({type: INITIALIZE_REGISTRY_KONNECTORS, konnectors: konnectors})
+let contextCache
+export const initializeRegistry = konnectors => {
+  return (dispatch, getState) => {
+    return contextCache
+        ? Promise.resolve(contextCache)
+        : cozy.client.fetchJSON('GET', '/settings/context')
+          .then(context => {
+            contextCache = context
+            const ctx = context.attributes
```

Comment:
It feels weird to have `context` and then `ctx`. When a variable name is too ambiguous, it is often because of an optimizable pattern. What about:
```js
const konnectorsToExclude = context.attributes && context.attributes['exclude_konnectors']
if (konnectorsToExclude && konnectorsToExclude.length) {
  // Avoid mutating konnectors
  return dispatch({type: INITIALIZE_REGISTRY_KONNECTORS, konnectors.filter(k => !konnectorsToExclude.includes(k.slug)})
}
 dispatch({type: INITIALIZE_REGISTRY_KONNECTORS, konnectors})
```